couchbase3/src/test/scala/org/apache/pekko/stream/connectors/couchbase3/CouchbaseSupport.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.pekko.stream.connectors.couchbase3
+
+import com.couchbase.client.core.io.CollectionIdentifier
+import com.couchbase.client.java._
+import com.couchbase.client.java.codec.JacksonJsonSerializer
+import com.couchbase.client.java.env.ClusterEnvironment
+import com.couchbase.client.java.json.JsonObject
+import com.fasterxml.jackson.databind.json.JsonMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.stream.connectors.couchbase3.CouchbaseSupport.bucketName
+
+import scala.collection.mutable
+
+object CouchbaseSupport {
```

Comment:
can we call this CouchbaseTestSupport? Unfortunately, there are people likely to raise issues if they think we have hardcoded a username and password for production code.

Reply:
yes